### PR TITLE
JPERF-625: Fix changelog

### PR DESCRIPTION
**Pull request by mzyromski-atlassian**
Fix changelog for #64

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -26,10 +26,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/infrastructure/compare/release-4.15.0...master
 ### Changed
-- Upgraded Docker to 19.03
+
 
 ### Fixed
-- Fixed compatibility with Ubuntu Eoan ([JPERF-625](https://ecosystem.atlassian.net/browse/JPERF-625))
+- Fix compatibility with Ubuntu Eoan ([JPERF-625](https://ecosystem.atlassian.net/browse/JPERF-625)).
+- Upgrade Docker to 19.03.
```
Comment (pbruski) [inline CHANGELOG.md:33]: Why isn't it "Changed"?

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,10 +26,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/infrastructure/compare/release-4.15.0...master
 ### Changed
```
Comment (dagguh) [inline CHANGELOG.md:28]:
```suggestion
```

File: CHANGELOG.md (hunk 1 above)
Comment (dagguh) [inline CHANGELOG.md:33]:
```suggestion
- Upgrade Docker to 19.03, which works with Ubuntu 16-19.
```

File: CHANGELOG.md (hunk 2 above)
Comment (mzyromski-atlassian) [reply CHANGELOG.md:28]: Oh, missed that. Thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (mzyromski-atlassian) [reply CHANGELOG.md]: I'll let @dagguh explain. :)

Comment (dagguh) [reply CHANGELOG.md]: Docker version isn't API, so none of the contracts changed.

**Pull request merged**